main.go

```diff
@@ -44,6 +48,27 @@ func createApp() *cli.App {
 	return app
 }
 
```

Comment:
missing comment above the method to document it

Reply:
Agree.